source/tutorials/fmt-cdiff.rst

```diff
@@ -0,0 +1,478 @@
+
+.. note:: Taxonomic classifiers perform best when they are trained based on your specific sample preparation and sequencing parameters, including the primers that were used for amplification and the length of your sequence reads. Therefore in general you should follow the instructions in :doc:`Training feature classifiers with q2-feature-classifier <../tutorials/feature-classifier>` to train your own taxonomic classifiers. We provide some common classifiers on our :doc:`data resources page <../data-resources>`, including Silva-based 16S classifiers, though in the future we may stop providing these in favor of having users train their own classifiers which will be most relevant to their sequence data.
+
+
+.. download::
+   :url: https://data.qiime2.org/2018.8/common/gg-13-8-99-515-806-nb-classifier.qza
+   :saveas: gg-13-8-99-515-806-nb-classifier.qza
+
+.. command-block::
+
+   qiime feature-classifier classify-sklearn \
+     --i-classifier gg-13-8-99-515-806-nb-classifier.qza \
+     --i-reads rep-seqs.qza \
+     --o-classification taxonomy.qza
+
+   qiime metadata tabulate \
+     --m-input-file taxonomy.qza \
+     --o-visualization taxonomy.qzv
+
+.. question::
+    Recall that our ``rep-seqs.qzv`` visualization allows you to easily BLAST the sequence associated with each feature against the NCBI nt database. Using that visualization and the ``taxonomy.qzv`` visualization created here, compare the taxonomic assignments with the taxonomy of the best BLAST hit for a few features. How similar are the assignments? If they're dissimilar, at what *taxonomic level* do they begin to differ (e.g., species, genus, family, ...)?
+
+Next, we can view the taxonomic composition of our samples with interactive bar plots. Generate those plots with the following command and then open the visualization.
+
+.. command-block::
+
+   qiime taxa barplot \
+     --i-table table.qza \
+     --i-taxonomy taxonomy.qza \
+     --m-metadata-file sample-metadata.tsv \
+     --o-visualization taxa-bar-plots.qzv
+
+.. question::
+    Visualize the samples at *Level 2* (which corresponds to the phylum level in this analysis), and then sort the samples by animations_gradient, and then by animations_gradient. What are the dominant phyla in before and after the FMT?
+
+
+.. _`fmt cdiff ancom`:
+
+Differential abundance testing with ANCOM
```

Comment:
Maybe we should skip ANCOM and use q2-gneiss, instead?

Reply:
q2-gneiss may be too much to show in this tutorial! gneiss is complicated enough / has enough steps that it really needs its own tutorial.

Reply:
@nbokulich, I think that is a fair point, but, is ANCOM any "easier" to explain? I think the beauty of docs like this is that you can cross-reference topics.